android/lib/pages/profile_page.dart

```diff
@@ -397,7 +454,7 @@ class _ProfilePageState extends State<ProfilePage> {
                                   mainAxisSize: MainAxisSize.min,
                                   children: [
                                     const Padding(padding: EdgeInsets.all(2.0)),
-                                    if (dropdown_selection.value! ==
+                                    if (dropdown_selection.value ==
```

Comment:
Another variable that needs to be renamed. Update to `dropdownSelection`.

Reply:
Renamed also